dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -135,6 +139,26 @@ public Entry(long id) {
         starredUsers = new HashSet<>(0);
     }
 
+
+    @JsonProperty("checker_id")
+    @ApiModelProperty(value = "The id of the associated checker workflow", position = 12)
```

Comment:
Looks like a duplicate position, not sure if relevant and hopefully we find a better solution soon too